rdr_service/resource/schemas/participant.py

```diff
@@ -58,37 +58,43 @@ class ModuleStatusSchema(Schema):
     external_id = fields.String(validate=validate.Length(max=120))
     response_status = fields.EnumString(enum=QuestionnaireResponseStatus)
     response_status_id = fields.EnumInteger(enum=QuestionnaireResponseStatus)
-
-    class Meta:
-        schema_id = SchemaID.participant_modules
-        resource_uri = 'Participant/{participant_id}/Modules'
-        # Exclude fields and/or functions to strip PII information from fields.
-        pii_fields = ()  # List fields that contain PII data.
-        pii_filter = {}  # dict(field: lambda function).
-
-
-class ConsentSchema(Schema):
-    """ Store participant consent information """
-    consent = fields.String(validate=validate.Length(max=80))
-    consent_id = fields.Int32()
-    consent_date = fields.Date()
+    questionnaire_response_id = fields.Int32()
+    consent = fields.Boolean()
     consent_value = fields.String(validate=validate.Length(max=80))
     consent_value_id = fields.Int32()
-    consent_module = fields.String(validate=validate.Length(max=80))
-    consent_module_authored = fields.DateTime()
-    consent_module_created = fields.DateTime()
     consent_expired = fields.String(validate=validate.Length(max=80))
-    consent_module_external_id = fields.String(validate=validate.Length(max=120))
-    consent_response_status = fields.EnumString(enum=QuestionnaireResponseStatus)
-    consent_response_status_id = fields.EnumInteger(enum=QuestionnaireResponseStatus)
+    non_participant_answer = fields.String(validate=validate.Length(max=60))
```

Comment:
We need to communicate any re-ogranization of fields for the BQ v_pdr_participant_module and v_pdr_participant_consent views especially to Scott;  I think he has queries that depend on those views that will break if we start removing fields from one and adding them to another (e.g.,have removed  consent_module, consent_module_authored, etc. here).

Should we leave those fields in here for now, until we get rid of the v_pdr_participant_consent view?